test/component_cloudwatch_test.go

```diff
@@ -12,10 +12,10 @@ import (
 func Test_cloudwatch(t *testing.T) {
 	setup(t)
 
-	pkgTest.Boot("test_configs/config.cloudwatch.test.yml")
-	defer pkgTest.Shutdown()
+	mocks := pkgTest.Boot("test_configs/config.cloudwatch.test.yml")
+	defer mocks.Shutdown()
 
-	cwClient := pkgTest.ProvideCloudwatchClient("cloudwatch")
+	cwClient := mocks.ProvideClient("cloudwatch", "cloudwatch").(*cloudwatch.CloudWatch)
```

Comment:
maybe we should rather have some constants for the component type

Reply:
Constants would not help here, because we need to use here the name of your mock.
You can call your mock whatever you want. It is the key of the mocks map in the configuration file.

Reply:
✅

Reply:
i got that however, we pass "cloudwatch" twice. One time for the component, which i would replace by the constant and one time as a name for the test component which could just stay as-is

Reply:
Check the new version. This problem is solved already with the `Clients` approach.